src/components/MultiHopTrade/components/TradeInput/TradeInput.tsx

```diff
@@ -288,14 +291,19 @@ export const TradeInput = (props: CardProps) => {
             </Stack>
             <Stack px={4}>
               <DonationCheckbox isLoading={isLoading} />
+              {rate && (
+                <ManualAddressEntry
+                  setIsManualAddressEntryValidating={setIsManualAddressEntryValidating}
+                />
+              )}
             </Stack>
-            <Tooltip label={activeQuoteStatus.error?.message}>
+            <Tooltip label={activeQuoteStatus.error?.message ?? activeQuoteStatus.quoteErrors[0]}>
```

Comment:
Not related to receive address, but gives us more info in the tooltip explaining why we can't proceed with a quote.